azure-nextgen-ts-functions-raw/python/HelloPython/__init__.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+import azure.functions as func
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info('Python HTTP trigger function processed a request.')
+
+    name = req.params.get('name')
+    if not name:
+        try:
+            req_body = req.get_json()
+        except ValueError:
+            pass
+        else:
```

Comment:
I think this is indented too far

Reply:
This is a shameless copy of https://github.com/pulumi/examples/blob/master/azure-ts-functions-raw/python/HelloPython/__init__.py. 

That said I think the intent is that the `else` block is invoked only if no exception is raised (as opposed to a finally block).